plugin/src/main/java/org/opennms/velocloud/requisition/PartnerRequisitionProvider.java

```diff
@@ -62,6 +70,63 @@ protected Requisition handleRequest(final Request request, final VelocloudApiCli
         final var requisition = ImmutableRequisition.newBuilder()
                 .setForeignSource(request.getForeignSource());
 
+        try {
+            for (var enterprise : client.getEnterprises()) {
+                final List<User> users = client.getUsers(enterprise.id);
+
+                //TODO: confirm  gateway roles are coming from enterprise users
+                //TODO: confirm Gateway AssetNodes destination.
```

Comment:
Should we move this to a new issue or do we have to decide in this PR?

Reply:
At this point I don't think is necessary to decide since we are just doing the bare bones
I'll create a jira for them